Files console-tools-0.2.3-old/screenfonttools/.unicode_stop.rej.swp and console-tools-0.2.3/screenfonttools/.unicode_stop.rej.swp differ
diff -ruN console-tools-0.2.3-old/screenfonttools/unicode_start console-tools-0.2.3/screenfonttools/unicode_start
--- console-tools-0.2.3-old/screenfonttools/unicode_start	2006-09-05 17:15:12.000000000 +0100
+++ console-tools-0.2.3/screenfonttools/unicode_start	2006-09-05 17:17:23.000000000 +0100
@@ -48,7 +48,11 @@
 
 NUM=`fgconsole --next-available`
 NUM=`expr ${NUM} - 1 `
-for vc in `seq 0 ${NUM}` 
+# This needs a better fix, but because unicode_{start,stop} are called
+# before getty starts the other consoles, if only one console set, then
+# set for the first 6 VTs, since  these are normally started by inittab
+[ $NUM -eq 1 ] && NUM=6
+for vc in `seq 1 ${NUM}` 
 do
   # Since 1.2.1 - for older kernels, use: echo -n -e '\033%8'
   /bin/echo -n -e '\033%G' > ${DEVICE_PREFIX}${vc}
diff -ruN console-tools-0.2.3-old/screenfonttools/unicode_stop console-tools-0.2.3/screenfonttools/unicode_stop
--- console-tools-0.2.3-old/screenfonttools/unicode_stop	2006-09-05 17:15:11.000000000 +0100
+++ console-tools-0.2.3/screenfonttools/unicode_stop	2006-09-05 17:17:13.000000000 +0100
@@ -29,7 +29,10 @@
 
 NUM=`fgconsole --next-available`
 NUM=`expr ${NUM} - 1`
-for vc in `seq 0 ${NUM}`
+# Cope with being called in S mode before getty starts the remaining
+# VTs:
+[ $NUM -eq 1 ] && NUM=6
+for vc in `seq 1 ${NUM}`
 do 
    /bin/echo -n -e '\033%@' > ${DEVICE_PREFIX}${vc}
 done
diff -ruN console-tools-0.2.3-old/screenfonttools/unicode_stop.rej console-tools-0.2.3/screenfonttools/unicode_stop.rej
--- console-tools-0.2.3-old/screenfonttools/unicode_stop.rej	1970-01-01 01:00:00.000000000 +0100
+++ console-tools-0.2.3/screenfonttools/unicode_stop.rej	2006-09-05 17:15:12.000000000 +0100
@@ -0,0 +1,18 @@
+***************
+*** 29,34 ****
+  
+  NUM=`fgconsole --next-available`
+  NUM=`expr ${NUM} - 1`
+  for vc in `seq 1 ${NUM}`
+  do 
+     /bin/echo -n -e '\033%@' > ${DEVICE_PREFIX}${vc}
+--- 29,37 ----
+  
+  NUM=`fgconsole --next-available`
+  NUM=`expr ${NUM} - 1`
++ # Cope with being called in S mode before getty starts the remaining
++ # VTs: 
++ [ $NUM -eq 1 ] && NUM=6
+  for vc in `seq 1 ${NUM}`
+  do 
+     /bin/echo -n -e '\033%@' > ${DEVICE_PREFIX}${vc}
